specification/cdn/resource-manager/Microsoft.Cdn/2016-10-02/cdn.json

```diff
@@ -343,6 +343,47 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Cdn/profiles/{profileName}/getSupportedOptimizationTypes": {
```

Comment:
@prakharsharma10 - Why isnt optimizationTypes a property on profiles resource type? So GET on /profiles/{profileName} contains this property in the response payload. If you dont want it to be included by defauls in the payload, You can consider exposing it using a query parameter $expand "supportedOptimizationTypes".
Do you want to ACL it explicitly through the POST action and hence you are doing it this way? Please help me understand the scenario.